### PR TITLE
Add: form for adding new data

### PR DESCRIPTION
**Pull request by Jedeo**
What does this PR do?
-Adds a dropDown menu for the form for users to add new data
Where should the reviewer start?
-please start with the `HTML` then move on to the `CSS` and `script` file
How should this be manually tested?
-browser
Any background context you want to provide?
-added a dark mode option for the application

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -104,6 +108,44 @@ <h2 class="steps-header">ACTIVITY</h2>
...
+              <section  class="hydration-data-inputs hidden">
+                <label for="numOfOunces">Number Of Ounces <input id="numOfOunces" type="text"></label>
+              </section>
+              <section  class="activity-data-inputs hidden">
+                <label for="minutesActive">Minutes Active <input id="minutesActive" type="text"></label>
+                <label for="numOfSteps">Number of Steps <input id="numOfSteps" type="text"></label>
+                <label for="flightsOfStairs">Flights Of Stairs <input id="flightsOfStairs" type="text"></label>
+              </section>
+              <section class="date-input hidden">
+                <input class="calender"
```
Comment (piperjarrett) [inline dist/index.html:138]: The calendar will always be shown correct?

**Review by piperjarrett — COMMENTED** [review]
Looks really good Jedeo! I like that you did the dark mode and light mode. Good job!

File: dist/index.html (hunk 1 above)
Comment (Jedeo) [reply dist/index.html:138]: it will show with the select they select in the dropDown

**Review by forsethnico — APPROVED** [review]
The entry form looks great! Maybe add a way to reset the form or go back to the options menu again to help the user.

**Pull request merged**